### ISSUE TITLE
Add a cooldown

### ISSUE THREAD

**Issue by dear-clouds** [v0idp/Mellow#39]
I would nice to being able to set a cooldown after every request sent to Ombi.

**Comment by v0idp** [comment]
Since you can set a request limit for your ombi user which will be used by the Mellow bot, there is no reason to add this feature.

**Comment by dear-clouds** [comment]
Yes i know but it's not working properly. The bot still says "Requested in Ombi" but there's nothing in Ombi since there's a limit set... So being able to set a cooldown to "!tv" and "!movie" would be really useful.

**Comment by v0idp** [comment]
I will keep this in mind and add it in the future
